### PR TITLE
tsd: add another properties of FileStream

### PR DESCRIPTION
**Pull request by Rwing**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -529,6 +529,20 @@ export interface Application extends EggApplication {
 
 interface FileStream extends Readable { // tslint:disable-line
   fields: any;
+
+  filename: string;
```
Comment (popomore) [inline index.d.ts]: 加一个 https://github.com/cojs/busboy 的链接

Comment (fengmk2) [inline index.d.ts]: 不需要 export ？

Comment (fengmk2) [inline index.d.ts]: ping @Rwing

Comment (popomore) [inline index.d.ts]: 啥 export

Comment (fengmk2) [inline index.d.ts]: 那就合并吧， @popomore 那个 url 可以你合并的时候手动加

Comment (Rwing) [inline index.d.ts]: 不好意思昨天没看到，不用 export。另外加这个 url 用途是？

**Pull request merged**